WcaOnRails/app/views/regional_organizations_mailer/notify_board_of_new_regional_organization_application.html.erb

```diff
@@ -0,0 +1,35 @@
+<h1>
+  <%= @user_who_applied.name %> has submitted an application for <%= @regional_organization.name %> to be acknowledged as a WCA Regional Organization.
+</h1>
+<p>
+  Here's the Regional Organization data:
+  <ul>
+    <li><b><%= t 'activerecord.attributes.regional_organization.name' %>:</b> <%= @regional_organization.name %></li>
+    <li><b><%= t 'activerecord.attributes.regional_organization.country' %>:</b> <%= @regional_organization.country %></li>
+    <li><b><%= t 'activerecord.attributes.regional_organization.website' %>:</b> <%= link_to "#{@regional_organization.website}", @regional_organization.website %></li>
+    <% if @regional_organization.logo.attached? %>
+      <li><b><%= t 'activerecord.attributes.regional_organization.logo' %>:</b></li>
+      <%= link_to(image_tag(@regional_organization.logo.variant(resize: "200x200")), rails_blob_url(@regional_organization.logo, disposition: "attachment")) %>
+    <% end %>
+    <li><b><%= t 'activerecord.attributes.regional_organization.email' %>:</b> <%= mail_to @regional_organization.email, "#{@regional_organization.email}" %></li>
+    <li><b><%= t 'activerecord.attributes.regional_organization.address' %>:</b> <%= @regional_organization.address %></li>
+    <% if @regional_organization.bylaws.attached? && @regional_organization.bylaws.previewable? %>
+      <li><b><%= t 'activerecord.attributes.regional_organization.bylaws' %>:</b></li>
+      <%= link_to(image_tag(@regional_organization.bylaws.preview(resize: "200x200")), rails_blob_url(@regional_organization.bylaws, disposition: "attachment")) %>
+    <% end %>
+    <li><b><%= t 'activerecord.attributes.regional_organization.directors_and_officers' %>:</b> <%= @regional_organization.directors_and_officers %></li>
+    <li><b><%= t 'activerecord.attributes.regional_organization.area_description' %>:</b> <%= @regional_organization.area_description %></li>
+    <li><b><%= t 'activerecord.attributes.regional_organization.past_and_current_activities' %>:</b> <%= @regional_organization.past_and_current_activities %></li>
+    <li><b><%= t 'activerecord.attributes.regional_organization.future_plans' %>:</b> <%= @regional_organization.future_plans %></li>
```

Comment:
Idea:
```erb
<% i18n_prefix = 'activerecord.attributes.regional_organization' %>
<!-- ... -->
<%= t "#{i18n_prefix}.future_plans" %>
```
Whatever you prefer =)

Reply:
I prefer the shorter (your) way :stuck_out_tongue: